pkg/gui/controllers/commit_message_controller.go

```diff
@@ -140,6 +156,10 @@ func (self *CommitMessageController) setCommitMessageAtIndex(index int) (bool, e
 }
 
 func (self *CommitMessageController) confirm() error {
+	if self.c.GocuiGui().IsPasting {
```

Comment:
I'd spell this out in a comment too i.e. the default keybinding for confirm is enter, and newlines are interpreted as enter presses when pasting.

Reply:
Added a condition and a comment in 8256ed219e7a, like for the other handlers.